### ISSUE TITLE
NullReferenceException when upgraded to Uno 4.9

### ISSUE THREAD

**Issue by HakanL** [unoplatform/uno.extensions#1534]
### Current behavior

Upgraded all Uno NuGet packages to 4.9 and when running it under Windows WinSDK I'm getting an error in the generated code for my main page:

```
partial class HomePage : Uno.Extensions.Navigation.IForceBindingsUpdate
				{

					ValueTask IForceBindingsUpdate.ForceBindingsUpdateAsync()
					{	
						this.Bindings.Update();
						return ValueTask.CompletedTask;
					}

				}
```

on this line: `this.Bindings.Update();`. `this.Bindings` is null. It works fine in GTK and Linux FB. Running the previously released version worked fine.

### Expected behavior

No crash. If I skip over the `this.Bindings.Update();` line then the application works fine.

### Works on UWP/WinUI

No

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

```
		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
		<PackageReference Include="Uno.Dsp.Tasks" Version="1.1.0" />
		<PackageReference Include="Uno.Extensions.Authentication.WinUI" Version="2.4.1" />
		<PackageReference Include="Uno.Extensions.Configuration" Version="2.4.1" />
		<PackageReference Include="Uno.Extensions.Hosting.WinUI" Version="2.4.1" />
		<PackageReference Include="Uno.Extensions.Logging.Serilog" Version="2.4.1" />
		<PackageReference Include="Uno.Extensions.Logging.WinUI" Version="2.4.1" />
		<PackageReference Include="Uno.Extensions.Navigation.Toolkit.WinUI" Version="2.4.1" />
		<PackageReference Include="Uno.Extensions.Navigation.WinUI" Version="2.4.1" />
		<PackageReference Include="Uno.Material.WinUI" Version="2.6.0" />
		<PackageReference Include="Uno.Resizetizer" Version="1.1.0" />
		<PackageReference Include="Uno.Toolkit.WinUI" Version="3.0.4" />
		<PackageReference Include="Uno.Toolkit.WinUI.Material" Version="3.0.4" />
		<PackageReference Include="Uno.WinUI" Version="4.9.17" />
		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.9.17" Condition="'$(Configuration)'=='Debug'" />
```

### Affected platforms

Windows

### IDE

Visual Studio 2022

### IDE version

17.6.2

**Comment by HakanL** [comment]
@nickrandolph Yep, that did it, thanks!

**Comment by nickrandolph** [comment]
@HakanL the fix has been merged - would you be so kind as to see if version 2.5.0-dev.66 fixes the issues (without the ForceUpdate attribute)? If you can confirm it, we'll backport this fix to the 4.9 release

**Comment by HakanL** [comment]
@nickrandolph 2.5.0-dev.66 worked great without the attribute, so confirmed fix!

**Comment by Youssef1313** [comment]
Thanks for confirming @HakanL. The backport was merged, so I'm closing the issue.

**Comment by nickrandolph** [comment]
Thanks @HakanL. The backport is available in version 2.4.2 of extensions if you want to update to that.

**Comment by HakanL** [comment]
@nickrandolph Confirming that 2.4.2 works!
